plugins/analysis/dashboard/frontend/src/app/models/ws/wsMsgType.ts

```diff
@@ -4,5 +4,13 @@ export enum WSMsgType {
     addNode = 2,
     removeNode = 3,
     connectNodes = 4,
-    disconnectNodes = 5
-}
+    disconnectNodes = 5,
+    Mana= 8,
+    ManaMapOverall,
+    ManaMapOnline,
+    ManaAllowedPledge,
+    ManaPledge,
+    ManaRevoke,
+    MsgManaDashboardAddress,
+    MsgReqManaDashboardAddress
```

Comment:
`MsgManaDashboardAddress` and `MsgReqManaDashboardAddress` are legit here, as they are msg types for communicating with the analysis dashboard. The other, mana related types are for communicating with a local dashboard, they don't belong here. Put them in WSMsgTypeDashboard or smthing. Let's not mix the analysis ws message enums with the dashboard ws message enums.